glue_jupyter/bqplot/scatter/layer_artist.py

```diff
@@ -349,18 +385,25 @@ def _update_visual_attributes(self, changed, force=False):
 
         if force or "visible" in changed:
             self.scatter_mark.visible = self.state.visible and self.state.markers_visible
-            self.line_mark.visible = self.state.visible and self.state.line_visible
+            if "linestyle" in changed:
+                self.line_mark_gl.visible = False
+                self.line_mark.visible = True
+                self.line_mark.visible = self.state.visible and self.state.line_visible
+            else:
+                self.line_mark_gl.visible = self.state.visible and self.state.line_visible
             self.density_mark.visible = (self.state.visible and self.state.density_map
                                          and self.state.markers_visible)
-            self.vector_mark.visible = self.state.visible and self.state.vector_visible
+            self.vector_lines.visible = self.state.visible and self.state.vector_visible
 
     def _update_scatter(self, force=False, **kwargs):
 
+        print( "_update_scatter triggered")
```

Comment:
also a flake8 failure here, but this looks like a leftover debugging print anyway?